.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Question
+about: Ask a Question about the project
+
+---
+
+**Checklist:**
+
+- [ ] I have checked stackoverflow for this question and there are no answers or the question does not exist
```

Comment:
@jayeb Are you happy to keep this requirement here - I believe this is consistent with what's described in `CONTRIBUTING.md`

Reply:
I think I feel the same way--I don't want to point people out to another forum if they've already found their way here. I'd rather treat this repo as the canonical source for questions and answers about this project. If you disagree strongly, though, I could probably be convinced to leave it.

Reply:
I don't feel strongly about this - the reason I brought it up is that I don't want to disagree with `CONTRIBUTING.md`. We're on the same page i.e. keep this repo the source for Q&A. I'll need to update the `CONTRIBUTING` to reflect this as well.